ansible.cfg

```diff
@@ -6,7 +6,7 @@ log_path = ~/ansible.log
 # We may consider ansible.builtin.junit
 callbacks_enabled = ansible.posix.profile_tasks
 display_args_to_stdout = True
-gathering = smart
+gathering = implicit
```

Comment:
Yes, I find this to be necessary because the cached facts with the host IPs get out of date and no longer represent the actual IPs.